squire_lib/src/tournament.rs

```diff
@@ -918,6 +918,59 @@ impl Tournament {
         }
         digest
     }
+
+    /// Generates a round slip in HTML
+    pub fn round_slips_html(self, css: String) -> String {
```

Comment:
A method that takes `self` consumes the whole object. We only need a reference to the tournament. Also, we don't need a whole string for the `css`. We only need a string slice, a `&str`.
```suggestion
    pub fn round_slips_html(&self, css: &str) -> String {
```